.gitlab-ci.yml

```diff
@@ -84,6 +84,7 @@ ubuntu-python3:cuda-9.0:build:
   extends: .build_template
 ubuntu-python3:cuda-10.1:build:
   extends: .build_template
+  timeout: 3h
```

Comment:
Why is this needed? I guess it shouldn‘t take longer than the other jobs.

Reply:
master is currently red because 2h aren't always enough